inc/namespace.php

```diff
@@ -349,6 +349,19 @@ function convert_internal_link( $url ) {
 			}
 			break;
 
+		case 'support':
+			$map = [
+				'new' => 'https://dashboard.altis-dxp.com/#/support/new',
+			];
+			$new_url = $map[ $host ] ?? null;
```

Comment:
Possibly should use a different variable name here if we decide to support other stuff in the future; e.g. if we had `support://tickets?status=open`, `$new_url` wouldn't make as much sense as a name.

Reply:
This is how the rest of the function works, `$new_url` is passed down to the filter at the end and returned from the function.

Reply:
oh, derp